### PR TITLE
DOC-9567 - Add scoped index management examples

### PR DESCRIPTION
**Pull request by maria-robobug**
- Use changes made in this sdk-common PR:
  couchbase/docs-sdk-common#47

- Add Index management examples with scopes and collections (new in 1.3)

### CODE REVIEW

File: modules/howtos/examples/QueryIndexManagerExample.scala [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    )
+    result match {
+      case Success(_)   =>
+      case Failure(err) => throw err
+    }
+
+    // Drop a secondary index
+   result = queryIndexMgr.dropIndex(
+      "travel-sample",
+			"tenant_agent_01_users_email",
```
Comment (osfameron) [inline modules/howtos/examples/QueryIndexManagerExample.scala]: odd indentation here

Comment (maria-robobug) [reply modules/howtos/examples/QueryIndexManagerExample.scala]: It's running away :D

Comment (osfameron) [reply modules/howtos/examples/QueryIndexManagerExample.scala]: eeek! 🏃 💨

**Review by osfameron — APPROVED** [review]
👍

**Pull request merged**